forester/tests/e2e_test.rs

```diff
@@ -335,17 +352,18 @@ async fn test_epoch_monitor_with_2_foresters() {
     assert!(total_processed > 0, "No items were processed");
 
     for tree in state_trees {
-        let queue_length =
-            fetch_queue_item_data(pool.get_connection().await, &tree.nullifier_queue)
-                .await
-                .unwrap()
-                .len();
+        let mut rpc = pool.get_connection().await.unwrap();
```

Comment:
can we map such errors instead of unwrapping, so that we can restart in case all connections drop for some reason?

Reply:
(There are a lot of unwraps in other places not commenting those explicitly.)

Reply:
I think it's safe enough. 
All the unwraps like this are only in the e2e test, forester maps [errors like this](https://github.com/Lightprotocol/light-protocol/blob/sergey/bb8-solana-rpc-pool/forester/src/epoch_manager.rs#L191).
And we also have a helper method for the case when all connections are busy: [get_connection_with_retry](https://github.com/Lightprotocol/light-protocol/blob/sergey/bb8-solana-rpc-pool/forester/src/rpc_pool.rs#L85)

Reply:
you are right unwraps are just in tests. We have a lot of unwraps in other places though (not related to this pr), but we can remove those in a different pr.